sql/20201027-is-demo.sql

```diff
@@ -0,0 +1,35 @@
+--- add is_demo to threads
+ALTER TABLE threads
+ADD COLUMN is_demo boolean;
+
+WITH demo_number AS (
+    SELECT DISTINCT(to_phone_number) as phone
```

Comment:
I believe you're selecting phone numbers here of people who texted with a demo line, right?

Consider a row where the to_phone_number is a person texting the line and the front_phone_number is the demo line

Reply:
there's the direction='INBOUND' condition below?